testsuite/openshift/objects/validating_webhook.py

```diff
@@ -0,0 +1,53 @@
+"""Kubernetes Validating Webhook object"""
+from typing import Any, Dict, TYPE_CHECKING
+
+from testsuite.objects import LifecycleObject
+from testsuite.openshift.objects import OpenShiftObject
+
+if TYPE_CHECKING:
+    # pylint: disable=cyclic-import
+    from testsuite.openshift.client import OpenShiftClient
+
+
+class ValidatingWebhook(OpenShiftObject, LifecycleObject):
+    """Represents Kubernetes ValidatingWebhook object"""
+
+    @classmethod
+    def create_instance(cls, openshift: 'OpenShiftClient', name: str, service_name: str, ca_bundle: str):
+        """Creates base instance"""
+        model: Dict[str, Any] = {
+            'apiVersion': 'admissionregistration.k8s.io/v1',
+            'kind': 'ValidatingWebhookConfiguration',
+            'metadata': {
+                'name': name,
+                'namespace': openshift.project
+            },
+            'webhooks': [
```

Comment:
it will not change I think

Reply:
If it doesnt need to change and is only useful for one test it doesn't need a class

Reply:
> If it doesnt need to change and is only useful for one test it doesn't need a class

This is still valid, in current state ValidatingWebhook class is not needed as it is hardcoded to Ingresses only and doesn't add any additional value, this can be a simple fixture and everything would work the same